src/coreclr-debug.ts

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import * as child_process from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+
+var _coreClrDebugDir: string;
+var _debugAdapterDir: string;
+var _channel: vscode.OutputChannel;
```

Comment:
It's OK, I can change them later when I do a full sweep through the code base. Essentially, in ES6 it's "let" for variables that can be assigned to and "const" for variables that can't be assigned to.

Don't sweat it.